bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3599,61 +3606,6 @@ spec:
   provider:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
-  relatedImages:
```

Comment:
@HarishH-DELL This section should be there. Please add it manually if deleted during building manifests